salt/metalk8s/addons/ui/deployed/ui-configuration.sls

```diff
@@ -54,7 +71,22 @@ Create metalk8s-shell-ui-config ConfigMap:
           config.yaml: |-
             apiVersion: addons.metalk8s.scality.com/v1alpha1
             kind: ShellUIConfig
-            spec: {}
```

Comment:
I think we also need the user group mapping here else we can't remove a group as well

Reply:
Debatable, I like that we can always assume all static users in Dex config will have the `metalk8s:admin` role (same as in Grafana config). If that's user-editable, then we can't update it when we change Dex config.